CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(cmake/add_dependency.cmake)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
 
 set(${PROJECT_NAME}_MAJOR_VERSION 02)
-set(${PROJECT_NAME}_MINOR_VERSION 06)
+set(${PROJECT_NAME}_MINOR_VERSION 08)
```

Comment:
Should be 7, not 8?

Reply:
No, this is intentional. The 02.07. is for the backport release based on DeviceAcces 02.08, the 02.08 is then the next free version in the master.
I will not squash merge the three commits and cherry-pick the one I need for the backport.